README.rst

```diff
@@ -73,6 +73,14 @@ or using `conda`::
 For detailed installation instructions, including instructions for installing
 from source, please read our `installation documentation <https://dipy.org/documentation/latest/installation/>`_.
 
+Python versions and dependencies
+--------------------------------
+
+DIPY follows the `Scientific Python`_ `SPEC 0 — Minimum Supported Versions`_
+recommendation as closely as possible, including the supported Python and
+dependencies versions.
+
+Further information can be found in :ref:`toolchain-roadmap`.
```

Comment:
I am not sure this reference will work (render correctly) on DIPY Github. 

I will go ahead and merge this PR but can you please check and do a follow up PR if it does not render correctly on https://github.com/dipy/dipy

Reply:
thank you!

Reply:
@skoudoro :+1: PR #3019.